src/init.cpp

```diff
@@ -1553,8 +1553,12 @@ bool AppInitMain(NodeContext& node, interfaces::BlockAndHeaderTipInfo* tip_info)
     ChainstateManager& chainman = *Assert(node.chainman);
 
     assert(!node.peerman);
-    node.peerman = PeerManager::make(*node.connman, *node.addrman, node.banman.get(),
-                                     chainman, *node.mempool, ignores_incoming_txs);
+    node.peerman = PeerManager::make(*node.connman, *node.addrman,
+                                     chainman, *node.mempool,
+                                     PeerManager::Options{
+                                         .banman = node.banman.get(),
```

Comment:
Why is the `banman` moved into the options?

Reply:
I thought it makes sense because the banman is optional but happy to move it out, no strong opinion on this. Let me know what you prefer. 

Will leave this unresolved for a bit to let others chime in as well.